banzai/utils/fits_utils.py

```diff
@@ -91,11 +91,7 @@ def table_to_fits(table):
     :param table: astropy table
     :return: fits BinTableHDU
     """
-    columns = [fits.Column(name=col.upper(), unit=str(table[col].unit),
-                           format=fits_formats(table[col].dtype),
-                           array=table[col]) for col in table.colnames]
-    hdu = fits.BinTableHDU.from_columns(columns)
-
+    hdu = fits.BinTableHDU(table)
```

Comment:
This change will affect a couple of things in the photometry HDU of reduced data, which probably don't matter too much, but the main one to note is that the column names will no longer be capitalized in (e.g. `XPEAK` will now be `xpeak`). 

Also, for some reason count units were output as `ct`, but they are now `count`; and columns without units used to have `unit = 'None'` but now they don't have any unit attribute at all.

Reply:
Ah okay, thanks. Curtis is there some way to test whether these changes will break the pipeline upon deployment? Or are e2e and units the only pre-deploy test options?

Reply:
So the fits record array (which these BinTableHDU.data are) do not have case sensitive keys. I think in general the FITS convention is not supposed to be case sensitive. If it becomes an issue, we can change the column names to be uppercase.